packages/teams-js/src/public/interfaces.ts

```diff
@@ -169,6 +169,71 @@ export enum FileOpenPreference {
   Web = 'web',
 }
 
+/**
+ * Action Types
+ */
+export enum ActionObjectType {
```

Comment:
Okay, so I figured out what the build error is about. All these types/enums/interfaces/etc. that you have added need to be exported from the interfaces file in the public/index.ts file. If they aren't exported from the SDK via the barrel file (the index.ts), then I think no documentation will get generated for them. 

When I synced your branch and explicitly exported them from the index.ts file, the errors got resolved.

Reply:
this is an interesting issue!

Reply:
this is extremely interesting, we should post this somewhere because I don't think I would have solved this on my own